src/server/scripts/ExilesReach/zone_exiles_reach.cpp

```diff
@@ -4114,6 +4296,616 @@ class spell_tutorial_health_dnt : public SpellScript
+        }
+
+        Creature* bunny = FindCreatureIgnorePhase(me, creatureString, 25.0f);
+
+        if (!bunny)
+            return;
+
+        Creature* prisoner = bunny->SummonCreature(_entry, 16.4271f, -2511.82f, 78.8215f, 5.66398f, TEMPSUMMON_MANUAL_DESPAWN);
+        _prisonerGUID = prisoner->GetGUID();
+    }
+
+    void JustDied(Unit* /*killer*/) override
+    {
+        if (Creature* prisoner = ObjectAccessor::GetCreature(*me, _prisonerGUID))
+            prisoner->AI()->SetData(SET_DATA_FREE_PRISONER, SET_DATA_FREE_PRISONER);
+    }
+
+    void JustEngagedWith(Unit* who) override
+    {
+        if (Player* player = who->ToPlayer())
+        {
+            Conversation* conversation = Conversation::CreateConversation(CONVERSATION_GEOLORD_AGGRO, player, *player, player->GetGUID(), nullptr, false);
+
+            if (!conversation)
+                return;
+
+            conversation->AddActor(CONVERSATION_ACTOR_GEOLORD, 0, me->GetGUID());
+            conversation->AddActor(CONVERSATION_ACTOR_LINDIE, 1, ObjectGuid::Empty);
+            conversation->AddActor(CONVERSATION_ACTOR_CORK, 2, ObjectGuid::Empty);
+            conversation->Start();
+
+            _events.ScheduleEvent(EVENT_CAST_EARTH_BOLT, 3s, 5s);
+            _events.ScheduleEvent(EVENT_CAST_UPHEAVAL, 10s, 15s);
+        }
+    }
+
+    void UpdateAI(uint32 diff) override
+    {
+        if (!UpdateVictim())
+            return;
+
+        _events.Update(diff);
+
+        while (uint32 eventId = _events.ExecuteEvent())
+        {
+            switch (eventId)
+            {
+                case EVENT_CAST_EARTH_BOLT:
+                {
+                    DoCastVictim(SPELL_EARTH_BOLT);
+                    _events.ScheduleEvent(EVENT_CAST_EARTH_BOLT, 10s, 12s);
+                    break;
+                }
+                case EVENT_CAST_UPHEAVAL:
+                {
+                    DoCastSelf(SPELL_UPHEAVAL);
+                    _events.ScheduleEvent(EVENT_CAST_UPHEAVAL, 10s, 15s);
+                    break;
+                }
+                default:
+                    break;
+            }
+        }
+
+        DoMeleeAttackIfReady();
+    }
+private:
+    EventMap _events;
+    ObjectGuid _prisonerGUID;
+};
+
+enum BriarpatchPrisoner
+{
+    EVENT_RUN_TO_PLAINS = 1,
+
+    SAY_GET_OUT_OF_HERE = 0
+};
+
+// This script is used by Cork Fizzlepop and Lindie Springstock for Down with the Quilboar quest
+struct npc_briarpatch_prisoner : public ScriptedAI
+{
+    npc_briarpatch_prisoner(Creature* creature) : ScriptedAI(creature) { }
+
+    void JustAppeared() override
+    {
+        me->SetDisableGravity(true);
+        me->SetTemplateRooted(true);
+        me->CastSpell(me, SPELL_NECROTIC_RITUAL_DNT);
+    }
+
+    void SetData(uint32 /*type*/, uint32 data) override
+    {
+        if (data == SET_DATA_FREE_PRISONER)
+        {
+            Position jumpto = { 19.5174f, -2513.75f, 74.0545f, me->GetOrientation() };
```

Comment:
move to const variable, i kinda assume the orientation does not change due to being rooted